aerialist/ros_node_script/spawn_script/spawn_dynamic_obstacles.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+import subprocess
```

Comment:
why don't you call this code (and all related code) directyly in simulator.py? we could have a seperate "gazebo / ros" class that wraps all the details of putting obstacles in the simulator.
is there any issues with that? let's discuss it
it is now being called by the launch file, which makes it hard to follow the code flow.